spec.md

```diff
@@ -137,6 +137,28 @@ attributes and the event data will be materialized. For example, in the case
 of a JSON serialization, the context attributes and the event data might
 both appear within the same JSON object.
 
+### Extension Attributes
+CloudEvent producers MAY include additional extension attributes within the
+event. This enables event producers, or middleware, to include additional
+metadata that might be used for any purpose in the processing of the
+CloudEvent, such as identifying or correlating event sources. See
+[CloudEvent Attributes Extensions](primer.md#cloudevent-attribute-extensions)
+for additional information concerning the use and definition of
+extensions.
+
+This specification places no restriction on the type or semantics of the
+extension attributes. Each definition of an extensions SHOULD fully
+define all aspects of the attribute - e.g. its name, semantic meaning
+and possible values.  New extension definitions SHOULD use a name that is
+descriptive enough to reduce the chances of name collisions with other
+extensions.
```

Comment:
New extension definition should use a name that does not collide with other extensions defined in the spec.md or extensions.md

Reply:
I tried to add some text to address this.